_posts/tsukori/2016-04-28-regextalk.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+author: tsukori
+title: "Regular Expressions Enhanced"
+---
+
+Professor Rex Dwyer from NC State came to give a talk about the re module on python and what he has done to modify it. He showed us real world application of python programming on DNA sequences which was really interesting. He showed us lookarounds which didn't really make a lot of sense to me? They seemed like they just cut strings in specific places without deleting the spot where it split it at, which was cool, but seemed like it did the same thing as the split function. 
```

Comment:
Lookarounds are like being able to selectively embed the split functionality within a regex, which makes them much more powerful and succinct for complex tasks.